src/pages/docs/releases/channels/index.md

```diff
@@ -40,15 +40,21 @@ As you add more channels, you'll notice that they are arranged in alphabetical o
 4. If you want to make this the default Channel for the project, click the **Default Channel** check-box.
 5. Design the [version rules](#version-rules) that will be used to enforce which versions of your packages are deployed to this channel.
 
-## Design the version rules {#version-rules}
+## Channel rules
 
-Version rules assist in selecting the correct versions of packages for the Channel.  They are only used when creating a release, either manually or via [project triggers](/docs/projects/project-triggers).
+Channels allow to you to configure rules to ensure that package versions and Git resources that meet specific criteria can be deployed using the channel.
+
+When creating a release for a channel with rules, an option can be configured on the project to allow the channel rules to be ignored. This option is disabled by default on new projects, but can be enabled using in project settings.
```

Comment:
`but can be enabled using in project settings.`

Should this be enabled in project settings or using the project settings? Also do we tend to refer to project settings as a proper noun or is it the project's settings?

Reply:
Ah good pickup, that was a "writing docs at the end of the day" special 😁. I'll take a look into how we refer to it in other places and follow.

Reply:
Looks like we "project settings" elsewhere so I'll go with that